spec/message-contracts/user_message_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.describe UserMessage do
+  context "when user data is in json" do
+    let(:user_json_data) { File.read("spec/message-contracts/user_message.json") }
```

Comment:
Small adjustment would be to put this let outside of contexts to dry it up. Minor speed improvement. And if you place it in fixtures dont forget to update here.